backend/package.json

```diff
@@ -37,7 +37,6 @@
 	"dependencies": {
 		"jsonwebtoken": "^8.5.1",
 		"mysql2": "^2.2.5",
-		"node-fetch": "^2.6.1",
```

Comment:
@ljoks doing this will break the backend we depend on his for oauth and canvas integration. Idk where why you thought to remove it.